agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -66,11 +66,17 @@ function collect_block {
 function collect_sos {
 	name=$(hostname -f)-pbench
 	debug_log "[$script_name]collecting sosreport"
-	sos_ver=`rpm -q sos | awk -F- '{print $2}' | awk -F. '{print $1}'`
-	if [[ "$sos_ver" == "3" ]]; then
+	read sos_ver sos_ver_minor <<< `rpm -q sos | awk -F- '{print $2}' | awk -F. '{print $1" "$2}'`
+	if [[ "$sos_ver" == 3 ]]; then
```

Comment:
Let's make this `-ge` 3 for now, and when 4 comes out, if it is a problem, we can amend.